content/en/altinity-kb-setup-and-maintenance/altinity-kb-zookeeper/zookeeper-cluster-migration-k8s-node-storage.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "ZooKeeper cluster migration when using K8s node local storage"
+linkTitle: "ZooKeeper cluster migration when using K8s node local storage"
+description: >
+  ZooKeeper cluster migration when using K8s node local storage
+---
+
+Describes how to migrate a ZooKeeper cluster when using K8s node-local storage such as static PV, `local-path`, `TopoLVM`.
+
+Requires HA setup (3+ pods).
+
+This solution is more risky than  due to reducing
+number of active consensus members but operationally simpler, when running with `clickhouse-keeper` can be
+performed gracefully so that quorum is maintained during the whole operation.
+
+
+1. Find the leader pod and note its name
+   1. To detect leader run `echo stat | nc 127.0.0.1 2181 | grep leader` inside pods
+2. Make sure the  ZK cluster is healthy and all nodes are in sync
+   1. (run on leader) `echo mntr | nc 127.0.0.1 2181 | grep zk_synced_followers` should be N-1 for N member cluster
+3. Pick the first **non-leader** pod and delete its `PVC`,
+   1. `kubectl delete --wait=false pvc clickhouse-keeper-data-0` -> status should be `Terminating`
+   2. Also delete `PV` if your `StorageClass` reclaim policy is set to `Retain`
+4. If you are using dynamic volume provisioning make adjustments (labels, taints, cordon) so that after pod delete the new one will be scheduled on the planned node
```

Comment:
Would love an example of this.

Reply:
added example using labels and taints, steps are highly depended on how your k8s workload placement is managed so there is large number of options how this could be done